test/OpenIddict.Server.Tests/OpenIddictServerExtensionsTests.cs

```diff
@@ -0,0 +1,403 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/openiddict/openiddict-core for more information concerning
+ * the license and the contributors participating to this project.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Localization;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using OpenIddict.Abstractions;
+using Xunit;
+
+namespace OpenIddict.Server.Tests
+{
+    public partial class OpenIddictServerExtensionsTests
+    {
+        [Fact]
+        public void AddServer_ThrowsAnExceptionForNullBuilder()
+        {
+            // Arrange
+            var builder = (OpenIddictBuilder)null;
+
+            // Act and assert
+            var exception = Assert.Throws<ArgumentNullException>(() => builder.AddServer());
+
+            Assert.Equal("builder", exception.ParamName);
+        }
+
+        [Fact]
+        public void AddServer_ThrowsAnExceptionForNullConfiguration()
+        {
+            // Arrange
+            var services = new ServiceCollection();
+            var builder = new OpenIddictBuilder(services);
+
+            // Act and assert
+            var exception = Assert.Throws<ArgumentNullException>(() => builder.AddServer(configuration: null));
+
+            Assert.Equal("configuration", exception.ParamName);
+        }
+
+        //[Fact]
+        //public void AddServer_RegistersAuthenticationServices()
```

Comment:
FYI, the authentication services are no longer registered by the generic server services (aka `AddServer()`) but by the ASP.NET Core host itself for layering reasons (`options.UseAspNetCore()`).

Reply:
Alright, removed!